cmake/multi_image.cmake

```diff
@@ -174,10 +174,13 @@ function(add_child_image_from_source)
   endif()
 
   # Pass information that the partition manager is enabled to Kconfig.
-  add_overlay_config(
-    ${ACI_NAME}
-    ${NRF_DIR}/subsys/partition_manager/partition_manager_enabled.conf
-    )
+  # Do it only if partition manager is enabled for the parent image.
+  if (CONFIG_PARTITION_MANAGER_ENABLED)
```

Comment:
how do this work with where there are multiple levels of "parents" ?

That is where a child-image is including extra child images, such as Bluetooth sample on nRF5340 which includes hci_rpmsg, which will then act as an extra parent for secure boot (b0n).